pytest_arraydiff/plugin.py

```diff
@@ -56,12 +113,20 @@ def read(filename):
         return fits.getdata(filename)
```

Comment:
No need to change this in this PR, but I think it would be nice to be able to compare multi extension fits in the future, or at least be able to loop through the extensions.